camp_fin/models.py

```diff
@@ -1020,15 +983,15 @@ def trends(self, since="2010"):
             SELECT
               months.year,
               months.month,
-              {table}.amount
+              COALESCE({table}.amount, 0) AS amount
             FROM (
               SELECT
                 DISTINCT DATE_PART('year', month) AS year,
                 GENERATE_SERIES(1, 12) AS month
               FROM {table}_by_month
               ORDER BY year, month
             ) months
-            LEFT JOIN (
```

Comment:
wait if we don't need to do LEFT JOIN here, then that suggests that every month we need is already in the generated table, in which case why do we need this subquery at all?

Reply:
if we don't need that left join here, then we should probably move the coalese logic up to the aggregate functions.

Reply:
Yeah, that would be awesome, because right now, the aggregate tables are sparse, so we can't distinguish between when there were $0 in transactions or we don't have data. That's why I did away with this left join - for the months on either end of the range, it reported $0 in transactions, but that might not actually be accurate.